addons/hr_holidays/models/res_partner.py

```diff
@@ -1,12 +1,20 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import api, models, fields
 from odoo.addons.mail.tools.discuss import Store
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
+    leave_date_to = fields.Date(compute="_compute_leave_date_to")
+    def _compute_leave_date_to(self):
+        for partner in self:
+            # in the rare case of multi-user partner, return the earliest
+            # possible return date
+            dates = partner.user_ids.mapped("leave_date_to")
+            partner.leave_date_to = min(dates) if dates and all(dates) else False
```

Comment:
Isn't this missing the code that was checking if the leave was `validate`?
Do we have a good reason to remove it?

Reply:
Yes, it was pointed out that this condition is part of the domain that compute `leave_date_to` (see [this comments](https://github.com/odoo/odoo/pull/202702#discussion_r2016774889) and [the domain](https://github.com/odoo-dev/odoo/blob/master-show-leave-avatar-did/addons/hr_holidays/models/hr_employee_base.py#L120))